docs/blog/posts/2025-07-03-bids-town-hall.md

```diff
@@ -0,0 +1,18 @@
+---
+date: 2025-07-03
+slug: "BIDS Virtual Town Hall 2025"
+author: anonymous
+categories:
+- town hall
+---
+
+Come share your views and learn more about how the BIDS project is developing
+
+The 2025 virtual BIDS Town Hall happening on Wednesday, July 16 at 11:00 AM EDT ([check your time here](https://www.worldtimebuddy.com/event?lid=8,6,5,12&h=8&sts=29210820&sln=8-9&a=preview&c=1381)).
+
+
+Hear more about our organizational updates, status updates from several of our BIDS extension proposals (BEPs), our community projects, and voice your thoughts about BIDS!

```

Comment:
```suggestion
Hear more about our organizational updates, status updates from several of our BIDS extension proposals (BEPs), our community projects, and voice your thoughts about BIDS!
```

to pacify remark

https://github.com/bids-standard/bids-website/actions/runs/16126416259/job/45504274326?pr=682#step:4:111